scripts/common.sh

```diff
@@ -42,7 +42,10 @@ set-sysroot() {
 
 cross-emerge() {
 	# Unfortunately, overlays can't have NFS layers.
+	# Also, overlayfs does not stack on overlayfs (e.g. inside docker containers).
 	case "$(stat -f -c %T "${EROOT}"/etc/portage)" in
+	overlayfs)
+		;&
```

Comment:
That syntax is very odd, I've never seen it before! Do you think `nfs|overlayfs)` would be clearer?

Reply:
That would also work - I wanted to  make it an explicit `case` "fall-through". But since we're not doing anything special for `overlayfs` _before_ we run the code for `nfs`,  `nfs|overlayfs)` will also do the job. Will update.